### PR TITLE
Add conversions

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:**
Internal ticket, to add conversions data 

**This PR will result in the following new package version:**

0.11.0

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

## Feature update
- We have added the following fields to each `pinterest_ads` end model:
  - `total_conversions`
  - `total_conversions_quantity`
  - `total_conversions_value_in_micro_dollar`
- In the event that you were already passing the above fields in via our [passthrough columns](https://github.com/fivetran/dbt_pinterest/blob/main/README.md#passing-through-additional-metrics), the package will dynamically avoid "duplicate column" errors.

> The above new field additions are 🚨 **breaking changes** 🚨 for users who were not already bringing in conversion fields via passthrough columns.

- Added more information about the difference in grains and their relationships in the [DECISIONLOG](https://github.com/fivetran/dbt_pinterest/blob/main/DECISIONLOG.md#pinterest-ads-grains).

## Under the hood
- Created `pinterest_ads_persist_pass_through_columns` macro to ensure that the new conversion fields are backwards compatible with users who have already included them via passthrough fields.
- Added integrity and consistency validation tests within `integration_tests` folder for the transformation models (to be used by maintainers only).

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [na] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [x] dbt run –vars (if applicable). --  to test backwards compatibility, see internal note

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

1. Backwards compat: tested as noted in internal ticket:

> 
> pinterest__ad_group_report_passthrough_metrics: 
>   - name: "total_conversions"   
>   - name: "total_view_checkout"
>     alias: "t_view_checkout"
> 

2. Prior metrics are not affected: see consistency tests
3. Conversion values aggregate correctly across grains that rollup: see horizontal integrity test `horizontal_sum_conversions`
4. Conversion values aggregate correctly from staging to final models: see vertical integrity test `vertical_sum_conversions`

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
     no new variables, but just checking that the existing passthrough column variable still works.
- [x] DECISIONLOG updates have been updated (if applicable) - added section about grains
- [x] Appropriate yml documentation has been added (if applicable) - added field definitions for new conversion fields

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [x] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+[PR #34](https://github.com/fivetran/dbt_pinterest/pull/34) includes the following updates:
+
+## Feature update
+- We have added the following fields to each `pinterest_ads` end model:
+  - `total_conversions`
+  - `total_conversions_quantity`
+  - `total_conversions_value_in_micro_dollar`
+- In the event that you were already passing the above fields in via our [passthrough columns](https://github.com/fivetran/dbt_pinterest/blob/main/README.md#passing-through-additional-metrics), the package will dynamically avoid "duplicate column" errors.
+
+> The above new field additions are 🚨 **breaking changes** 🚨 for users who were not already bringing in conversion fields via passthrough columns.
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:11]: Small request to remove emojis

```suggestion
> The above new field additions are **breaking changes** for users who were not already bringing in conversion fields via passthrough columns.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
+# dbt_pinterest v0.11.0
+[PR #34](https://github.com/fivetran/dbt_pinterest/pull/34) includes the following updates:
+
+## Feature update
+- We have added the following fields to each `pinterest_ads` end model:
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:5]: Maybe we can add a bit more fanfare here to highlight native support for conversions.

File: packages.yml [hunk 3]
```diff
@@ -1,3 +1,7 @@
 packages:
-  - package: fivetran/pinterest_source
-    version: [">=0.10.0", "<0.11.0"]
+  # - package: fivetran/pinterest_source
+  #   version: [">=0.11.0", "<0.12.0"]
+
+  - git: https://github.com/fivetran/dbt_pinterest_source.git
```
Comment (fivetran-jamie) [inline packages.yml:5]: switch

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

File: integration_tests/dbt_project.yml [hunk 4]
```diff
@@ -16,20 +16,31 @@ vars:
...
 
+  pinterest__ad_group_report_passthrough_metrics: 
+    - name: "total_conversions"
+    - name: "total_view_checkout"
+      alias: "t_view_checkout"
+    - name: "total_conversions_quantity"
+      alias: "total_conversions_quantity_aliased"
+
+# models:
+#   +schema: "pinterest_ads_{{ var('directed_schema','dev') }}"
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply integration_tests/dbt_project.yml]: updated

**Pull request merged**